wasm2c/wasm-rt-impl.c

```diff
@@ -46,23 +46,24 @@ typedef struct FuncType {
 } FuncType;
 
 #if WASM_RT_MEMCHECK_SIGNAL_HANDLER
-bool g_signal_handler_installed = false;
-char* g_alt_stack;
+static bool g_signal_handler_installed = false;
```

Comment:
Given the way the signal handler works it could be more meaningful for this to remain an extern global -- it's not like somebody can link two wasm-rt-impl.o's together in the same process and have the different signal handlers play nicely together.

Reply:
Perhaps.. although I'm not sure that a symbol collision is the best way to prevent that... not sure what is though.

Reply:
I think I'll leave this one as static for now.   Double linking of wasm-rt-impl.o will already fail because of all the public symbols would be duplicately defined.